test/powershell/Host/Startup.Tests.ps1

```diff
@@ -0,0 +1,120 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+Describe "Validate start of console host" -Tag CI {
```

Comment:
After the startup perf PR was merged, we can avoid 16 assemblies from this list on Windows. Can you please update this expected list?

Reply:
Can you list out the assemblies?

Reply:
Looked at your PR and removed the managed assemblies